source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -20,29 +20,67 @@ There are a some limitations for this kind of use case:
 -  The imapc code doesn't support many IMAP features. Most importantly
    SEARCH isn't supported, which may result in lower performance.
 
-
 Setting up user-shared folders
 ------------------------------
 
 You'll need to setup master user logins to work for all the users. The
 logged in user becomes the master user. This way the ACLs are applied
-correctly.
+correctly. It is supported to access shared mailboxes of users which live on other
+backends via the IMAP protocol. For this purpose shared mailboxes in a cluster
+must be accessed via another namespace which has ``location`` configured to use
+``imapc``.
+
+Mailboxes in the shared namespace can be accessed using the given namespace
+prefix. For example the user ``someone1`` could have access to a
+mailbox(``shared_mailbox``) owned by the user ``someone2``. To access mails
+in this mailbox ``someone1`` would access a mailbox with the name
+``shared/someone2/shared_mailbox`` the permissions which are given to
+``someone1`` on this mailbox are determined by :ref:`acl` s.
+
 
 ::
 
    namespace {
     type = shared
     prefix = shared/%%u/
-    location = imapc:~/shared/%%u/ # cache for shared indexes
+    list = children
+    location = imapc:
    }
+
+Additionally imapc must be configured accordingly on the backends:
+ * :ref:`setting-imapc_master_user` must be set to ``%u`` e.g. the logged in
+   user becomes imapc master user
+ * :ref:`setting-imapc_password` must be set to the master password which is
+   configured on all backends and directors
+ * :ref:`setting-imapc_host` must be configured to a directors address
+
+
+::
+
    imapc_host = director-ip
    imapc_master_user = %u
    #imapc_user = # leave this empty. It'll be automatically filled with the destination username.
    imapc_password = master-secret
 
-   plugin {
-     acl_shared_dict = fs:posix:prefix=/nfs/shared-acls/
-   }
+As mentioned earlier the :ref:`acl` plugin must be loaded and configured
+accordingly:
+
+::
+
+    plugin {
+      acl = vfile
+      # Added in 2.3.15
+      acl_ignore_namespace = shared/*
+      acl_shared_dict = $your_prefered_shared_dict
```

Comment:
I think an example would still be good. Although is there any dict that would currently work across backends without NFS?..

Reply:
Or could it just point to "Shared mailbox listing" which has some examples already?..

Reply:
Oh, it would be better below. So this is fine here I guess.